Don't fetch the model from aws s3

dbason opened the pull request.

tybalex reviewed (approved):
lgtm

sanjay920 commented on nulog-inference-service/start-nulog-inference.py:
```diff
@@ -376,17 +345,19 @@ async def schedule_update_pretrain_model(logs_queue):
+        init_model_task = loop.create_task(get_pretrain_model(logs_queue))
+        model_loaded = loop.run_until_complete(init_model_task)
+        if not model_loaded:
+            os._exit(1)
```
I suggest `sys.exit()` over `os._exit()`. any reason why you went with the latter?

dbason replied:
Only because it lines up with what we do in go.  I have no strong preference.

sanjay920 reviewed (commented):
see comment

dbason commented:
Just pushed a couple of minor changes addressing your comment @sanjay920 , and removing the reload signal.  The signal isn't needed because when we first extract the model we haven't started inferencing

AmartC reviewed (approved):
Tested out the changes and everything is running smoothly. LGTM
